cli/src/core/config/viper.go

```diff
@@ -0,0 +1,6 @@
+package config
+
```

Comment:
(Optional)
I disagree with the name of this file, as much as this file is being used by viper it's actually just being used to store the physical paths to all files that relate to configs.
ie. if we change the tool we use to process these paths does that now mean we need to rename this file?
Perhaps "paths" or something to that effect would be more descriptive